tests/backstop_data/backstop.tpl.json

```diff
@@ -2,8 +2,8 @@
   "id": "Civicase 5.1",
   "viewports": [{
     "name": "desktop",
-    "width": 1920,
-    "height": 3000
+    "width": 1680,
```

Comment:
These changes are from previous PR right? I think the branch is not properly rebased.